icons/soup.svg

```diff
@@ -1,4 +1,13 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
-  <path fill="#fff" d="M0 0h24v24H0z"/>
-  <path stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 21c6.5 0 9-4.03 9-9H3c0 4.97 2.5 9 9 9Zm7.5-9L22 6M6.247 3c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618m4.512-5c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618m4.512-5c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618M7 21h10"/>
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 21c6.5 0 9-4.03 9-9H3c0 4.97 2.5 9 9 9Zm7.5-9L22 6M6.247 3c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618m4.512-5c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618m4.512-5c.274.106.808.526.748 1.358-.059.833-.93 1.201-.99 2.024-.055.78.334 1.242.73 1.618M7 21h10"/>
```

Comment:
Can you break this up into its separate paths?

Unfortunately some SVG editors end up connecting segments of compound paths such as this, so it's safer to keep them separate.

E.g. stuff like this sometimes happen:
![image](https://user-images.githubusercontent.com/17746067/214272516-f944d20b-99bb-43a7-8a52-bfa6ca864d08.png)